samples/server/petstore/rust-server/src/client.rs

```diff
@@ -1649,7 +1628,7 @@ impl Api for Client {
                 0 => {
 
 
-                    Ok(CreateUsersWithListInputResponse::SuccessfulOperation)
+                    Ok(CreateUsersWithListInputResponse::Status0)
```

Comment:
This file contains the errant `Statusxxx` I mentioned earlier.

In any case, this `Status0` worries me - it looks as if we're not handling the `default` case (see https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#responses-object).